lib/v2/datasource/remote/api/seeds_history_repository.dart

```diff
@@ -0,0 +1,34 @@
+import 'package:async/async.dart';
+import 'package:http/http.dart' as http;
+import 'package:seeds/v2/datasource/remote/api/network_repository.dart';
+import 'package:seeds/v2/datasource/remote/model/seeds_history_model.dart';
+
+class SeedsHistoryRepository extends NetworkRepository {
+
+  Future<Result<dynamic>> getNumberOfTransactions(String userAccount) {
+    print('[http] get seeds seeds history for account: $userAccount ');
+
+    final String request = createRequest(
+      code: "histry.seeds",
+      scope: "histry.seeds",
+      table: "totals",
+      tableKey: "",
+      lowerBound: '$userAccount',
```

Comment:
I think it will also work if you remove
tableKey (IDK what that even is)
indexPosition
keyType

Reply:
Especially keyType - if that works as "" then it may not work on all nodes... that's undocumented, we should not use that.

If we do define keyType it needs to be a valid type such as uint64 or whatever. (don't quote me on this, there's several valid key types defined in the API - need to look up the exact values that are valid)

Reply:
I think it should be like this (remove unnecessary params)

```
final String request = createRequest(
       code: "histry.seeds",
       scope: "histry.seeds",
       table: "totals",
       lowerBound: '$userAccount',
       upperBound: '$userAccount',
       limit: 1,
     );
```

Reply:
Hi, Thanks for the review, I made the new changes